sys/net/gnrc/application_layer/tftp/gnrc_tftp.c

```diff
@@ -53,7 +53,7 @@ static kernel_pid_t _tftp_kernel_pid;
 #endif
 
 #define MIN(a, b)                   ((a) > (b) ? (b) : (a))
-#define ARRAY_LEN(x)                (sizeof(x) / sizeof(x[0]))
+#define ARRAY_LEN(x)                ARRAY_SIZE(x)
```

Comment:
Why not remove `ARRAY_LEN` and replace its use with `ARRAY_SIZE`?

Reply:
Good catch!